arch/arm/boot/dts/overlays/README

```diff
@@ -1762,6 +1762,21 @@ Params: oscillator              Clock frequency for the CAN controller (Hz)
 
         interrupt               GPIO for interrupt signal
 
+Name:   mcp25xxfd-spiX-Y
+Info:   Configures the MCPXXFD CAN controller family on spiX.Y
+Load:   dtoverlay=mcp25xxfd-spiX-Y,<param>=<val>
+Params: oscillator              Clock frequency for the CAN controller (Hz)
+
+        spimaxfrequency         Maximum SPI frequence (Hz)
```

Comment:
There's a convention to call this property `speed`.

Reply:
ok